source/mainnet/smart-contracts/guides/compile-module.rst

```diff
@@ -73,11 +89,17 @@ If you want a smaller code footprint, enable the :ref:`wee_alloc feature <setup-
    Note that even with ``--release`` set, the produced Wasm module includes
    debug information.
 
-Running the ``cargo concordium build`` command will produce a smart contract module which can be found
+Non-verifiable builds
+---------------------
+
+Omitting the ``--verifiable`` option from the build command will make a build on the host machine.
+This is typically not verifiable since the Rust toolchain is affected by the build environment.
+However, for local development, non-verifiable builds can be useful.
```

Comment:
I guess closed sourced smart contracts will also want to avoid using it.

Reply:
Yes, but I don't want to mention those. It's fundamentally at odds with decentralization to do things on the blockchain and keep the source private .